src/libraries/Native/native-binplace.proj

```diff
@@ -10,7 +10,7 @@
 
   <!-- Ordering matters! Overriding GetBinPlaceItems and Build targets after the Sdk import. -->
   <Target Name="GetBinPlaceItems">
-    <ItemGroup>
+    <ItemGroup Condition="'$(BinPlaceTestRuntimePack)' != 'true'">
```

Comment:
Can we avoid duplicating the `ItemGroups` by doing something like:

```xml
<ItemGroup>
  <NativeBinPlaceItems Include="$(NativeBinDir)*.dll" />
  ...
  <NativeBinPlaceItems Include="$(NativeBinDir)*.dwarf" />
  
  <BinPlaceItem Condition="'$(BinPlaceTestRuntimePack)' != 'true'" Include="@(_nativeItems)" />
  <FileWrites Include="@(NativeBinPlaceItems)" />
 />
```

Then in the BinPlaceTargetFramework settings you're adding instead of using:  `NativeRuntimePackBinPlaceItem` as the `ItemName` you could change that to be `NativeBinPlaceItems`... that way you simplify it a lot.